homestead.md

```diff
@@ -514,6 +524,72 @@ Homestead uses the Nginx web server by default. However, it can install Apache i
 
 Homestead includes the Postfix mail transfer agent, which is listening on port `1025` by default. So, you may instruct your application to use the `smtp` mail driver on `localhost` port `1025`. Then, all sent mail will be handled by Postfix and caught by Mailhog. To view your sent emails, open [http://localhost:8025](http://localhost:8025) in your web browser.
 
+<a name="database-snapshots"></a>
+### Database Snapshots
```

Comment:
I feel like this section never gives me a step-by-step guide of how to actually use the feature.

Reply:
I'm happy to take a shot at this once we have the new base box out and I can play around with it.